notifications_utils/jinja_templates/email_template.jinja2

```diff
@@ -16,6 +16,8 @@
     }
     body { margin:0 !important; }
     div[style*="margin: 16px 0"] { margin:0 !important; }
+    .force-white-bg { background-color: #fff; background: linear-gradient(#fff, #fff); background-image: url("https://assets.notification.canada.ca/1x1-ffffff.png"); }
+    [data-ogsc] .force-white-bg { background-color: #fff; }
```

Comment:
Forcing a white background is incredibly difficult so we use a mix of tricks, like a fake linear background or a 1x1 white image as a background image

Reply:
🤯